src/models.py

```diff
@@ -29,11 +22,7 @@ class SuperUserRoles(str, Enum):
 
 class User(BaseModel, validate_assignment=True):
     user_id: PositiveInt
-    username: str = Field(
-        min_length=2,
-        pattern=r"^[a-zA-Z]*$",
-        description="Username must contain only letters and be at least two characters long.",
-    )
+    username: str
```

Comment:
The validation is in the field_validator in line 30 :) To be honest I don't remember exactly why I switched to the field validator, but I think it was in order to have a clearer error message in case of validation error.

Reply:
Ah ok, great then! :)